src/main/resources/META-INF/resources/primefaces/jquery/jquery.ui.timepicker.js

```diff
@@ -1572,12 +1578,20 @@
 		  return;
 		}
 
-		var tzoffset = $.timepicker.timezoneOffsetNumber(tp_inst.timezone);
+		/* PrimeFaces Customization
+	    var tzoffset = $.timepicker.timezoneOffsetNumber(tp_inst.timezone);
```

Comment:
This customization is originated from https://github.com/trentrichardson/jQuery-Timepicker-Addon/issues/281 if we look at https://github.com/primefaces/primefaces/blob/1926c7e974cc0b3f692345823be8d4491a394f51/src/main/resources/META-INF/resources/primefaces/jquery/jquery-plugins.js#L12231, but it seems to be fixed in `1.6.2` and `1.6.3` natively if you look at this commit: https://github.com/trentrichardson/jQuery-Timepicker-Addon/commit/60b93e495387488a890c7e5aa2e834deb3fdb17a